requirements.txt

```diff
@@ -21,6 +21,7 @@ ipython==5.3.0
 numpy>=1.10.1
 pexpect>=4.0.1
 psutil>=4.2.0
+pycurl==7.19.0
```

Comment:
Shouldn't this just be dropped if it anyway comes from fts3?

Reply:
I wasn't aware till just now, but on xenial the fts3 requirement of pycurl does not compile and automatically pics a higher version... now I reverted to trusty

Reply:
also on trusty you get
```
ERROR: fts3-rest 3.8.4 has requirement pycurl==7.19.0, but you'll have pycurl 7.43.0.2 which is incompatible.
```

Reply:
Completely removing the line solves it